content/en/project/federated_heterogeneous/index.md

```diff
@@ -0,0 +1,89 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+
+title: "Ensembling Heterogeneous Models for Federated Learning"
+summary: ""
+authors: [Kundjanasith Thonglek]
+tags: []
```

Comment:
Add some tags.
The tag will be appeared as an keyword to filter the research topics.
How about just add:
tags: ["Machine Learning"]